src/components/browse/datasetlist/datasetheader/datasetdetails/datasetDetails.tsx

```diff
@@ -14,21 +14,30 @@ const DatasetDetails = (props: dataSetDetailsProps) => {
   let hit = props.hit;
 
   return (
-    <div className="fs-8">
+    <div className="fs-9">
       <Row>
-        <Col lg md sm xl xs xxl="10">
+        <Row className="pe-0">
+          <Col>
           <p className="my-0">
             <span className="fw-bold">Dataset ID:&nbsp;</span>
-            {hit.content.accession}
-
-            <br />
-            <span className="fw-bold">Description:&nbsp;</span>
-            {hit.content.description}
+            <span style={{userSelect: "all"}}>{hit.content.accession}</span>
```

Comment:
Can we use this {{ userSelect: "all" }}for the accordion header to select the text?

Reply:
Yes but it doesn't work well because the header is a button.